projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -167,6 +171,14 @@ export const Publicize = withModuleSettingsFormHelpers(
 								{ shouldShowChildElements && hasSocialImageGenerator && (
 									<SocialImageGeneratorSection />
 								) }
+								{ isActive &&
+								isLinked &&
+								useAdminUiV1 &&
+								! this.props.isSavingAnyOption( 'publicize' ) ? (
```

Comment:
Could this be?

```suggestion
								{ shouldShowChildElements &&
								useAdminUiV1 && (
```

I'm not sure why we're not checking `isLinked` for the other sub-options

Reply:
Yeah, I agree. Didn't want to refactor in this PR.